src/go/util/token_generator_test.go

```diff
@@ -15,21 +15,32 @@
 package util
 
 import (
+	"io/ioutil"
+	"os"
 	"strings"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/esp-v2/src/go/util/testdata"
 )
 
 func TestGenerateAccessToken(t *testing.T) {
+	tmpFile, err := ioutil.TempFile(os.TempDir(), "TestGenerateAccessToken-")
```

Comment:
I suggest unit tests do not use files. This will be difficult to import into google3 because they need to use `runfiles` format.

Reply:
The functional code does read from file, so unit test based on file is necessary.
Is it very difficult to do this in copybara? If so, since this piece of code is not used by gateway, so maybe just skip  in copybara?

Reply:
Previously, this test was working because it did not test the `GenerateTokenFromFile` function. That was written so the file can be decoupled from the unit test. Can you re-write it for that? Or just test `generateAccessToken` and `activeAccessToken`?

Otherwise, you will need to hook in with the integration test framework 
 `tests/env/platform/files.go` for this to work in google3. There's no easy way to copybara it otherwise.

Reply:
put the file path in `tests/env/platform/files.go` and create/write/read/delete in the runtime. Do you think it works in google3?

Reply:
Can you remove the creating/deleting of the file? It probably won't work in google3. Why not just write the file and check it into our repo and use it for the test? Just reading the file will work (see other tests that use `platform` to read files).

Also name the file `FakeServiceAccount` or something like that. The file doesn't have an access token in it.

Reply:
Done